ui/design/src/components/TopBar/index.tsx

```diff
@@ -434,8 +450,52 @@ const Topbar: React.FC<ITopbarProps> = props => {
           onDashboardClick={onDashboardClick}
           modalSlotId={modalSlotId}
         />
-      )}
-    </TopbarWrapper>
+      );
+    }
+  };
+
+  return (
+    <Router>
+      <Switch>
+        <Route
+          path="/auth-app/*"
```

Comment:
can you add some tests covering this special case? You can use .skip for now until the code is merged into the next branch but at least we should have them written